src/MTopJetPreSelectionModule3Jets.cxx

```diff
@@ -0,0 +1,204 @@
+#include <iostream>
+#include <memory>
+
+#include <UHH2/core/include/AnalysisModule.h>
+#include <UHH2/core/include/Event.h>
+#include <UHH2/core/include/Selection.h>
+
+#include <UHH2/common/include/NSelections.h>
+#include <UHH2/common/include/LumiSelection.h>
+#include <UHH2/common/include/TriggerSelection.h>
+#include <UHH2/common/include/JetCorrections.h>
+#include <UHH2/common/include/ObjectIdUtils.h>
+#include <UHH2/common/include/JetIds.h>
+#include <UHH2/common/include/TopJetIds.h>
+#include <UHH2/common/include/TTbarGen.h>
+#include <UHH2/common/include/Utils.h>
+#include <UHH2/common/include/AdditionalSelections.h>
+#include "UHH2/common/include/YearRunSwitchers.h"
+
+// Hists
+#include <UHH2/common/include/ElectronHists.h>
+#include <UHH2/common/include/EventHists.h>
+#include <UHH2/common/include/MuonHists.h>
+#include <UHH2/common/include/JetHists.h>
+#include <UHH2/common/include/TTbarGenHists.h>
+#include <UHH2/MTopJet_UL/include/MTopJetHists.h>
+//
+#include <UHH2/MTopJet_UL/include/ModuleBASE.h>
+#include <UHH2/MTopJet_UL/include/RecoSelections.h>
+#include <UHH2/MTopJet_UL/include/GenSelections.h>
+
+using namespace std;
+
+class MTopJetPreSelectionModule3Jets : public ModuleBASE {
```

Comment:
Please change the name of the module to `MTopJetPreSelection`. So the module name itself but also the file name. You can use `git mv <fileold> <filenew>`